docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Runs SQL LRS with PostgreSQL - Provided for demonstration purposes only!
+# To run: docker compose up
+# See the Docker Compose docs for more info: https://docs.docker.com/compose/
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: my_db_user
+      POSTGRES_PASSWORD: my_db_password
+      POSTGRES_DB: lrsql_db
+  lrs:
+    # build: . # switch to this for active dev
+    image: yetanalytics/lrsql:latest
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
+      LRSQL_API_SECRET_DEFAULT: my_secret
+      LRSQL_ADMIN_USER_DEFAULT: my_username
+      LRSQL_ADMIN_PASS_DEFAULT: my_password
+      LRSQL_DB_HOST: db
+      LRSQL_DB_NAME: lrsql_db
+      LRSQL_DB_USER: my_db_user
+      LRSQL_DB_PASSWORD: my_db_password
```

Comment:
The env vars here (except for `LRSQL_DB_NAME`) differ from the ones in the doc. Also, the env vars here do not set a schema, whereas the ones in the doc do.

Reply:
The docker example is not the compose file. They are different ways to launch.

As for the env vars I can make them match except for LRSQL_DB_HOST - different because that's how hostnames work. In Compose `db` is the hostname of the database service (look above in the file)

For the schema, we cannot set our own without additional scripting to create the schema. If we omit it everything works fine, so I do!

Reply:
Yeah, I want the doc example and the compose file to line up as much as possible. Obviously I don't want them to line up to the point it breaks everything.

In the schema case, it's probably better to omit the schema from the doc example than to keep it - less complexity overall in addition to lining up with the compose file.

Reply:
See https://github.com/yetanalytics/lrsql/pull/169/commits/c97bcd9f6d85cfee15d13bf03303013c34331302

That's as lined up as they get without breaking. I removed schema.